modules/showBook.js

```diff
@@ -0,0 +1,37 @@
+import { addSection } from '../index.js'; /* eslint-disable-line */
+import removeBook from './removeBook.js'; /* eslint-disable-line */
+
+export function showBook(bookArray) { /* eslint-disable-line */
```

Comment:
- [ ] As this is ES6 it is good practice to use arrow functions 

![image](https://user-images.githubusercontent.com/75126481/165133076-9287c4c1-0a08-4f30-afe3-980f4ccd3a54.png)